templates/errors.rb

```diff
@@ -23,7 +23,7 @@
   Net::SMTPAuthenticationError,
   Net::SMTPServerBusy,
   Net::SMTPUnknownError,
-  TimeoutError
+  Timeout::Error
```

Comment:
I think it makes sense to ignore this, since the rest of the file doesn't use that style.